src/ProjectService.Business/Commands/Project/EditProjectCommand.cs

```diff
@@ -91,23 +95,22 @@ public OperationResultResponse<bool> Execute(Guid projectId, JsonPatchDocument<E
         {
             _validator.ValidateAndThrowCustom(request);
 
-            GetProjectFilter filter = new GetProjectFilter { ProjectId = projectId };
-
-            DbProject dbProject = _repository.Get(filter);
+            DbProject dbProject = _projectRepository.Get(new GetProjectFilter { ProjectId = projectId, IncludeUsers = true });
```

Comment:
What for do you add filters?

Reply:
In order to get the desired values ​​from the repository